### DISCUSSION TITLE
Add support for custom LUA reference file paths in mods folder

### DISCUSSION THREAD

**Discussion by SenorRagequit** [exorstudios/riftbreaker-tools#132] [Wishlist] [answered]
**Wish**
---
Support for custom LUA reference file paths in mods folder.

**Why**
---
This change will allow faster mod tests that use custom LUA with multiple linked references over wide variety of files and folders. It seems that custom LUA with more than two custom file path references refuse to load when put in mods folder. They can load just fine when zipped and put in packs folder but this takes time to test every change. The packs folder method is also bit cumbersome.

**Comment by lukaasm** [comment]
I need an example of what doesn't work.

There is no limit on how many "require" clauses you can use in the script. Or you mean something else as "reference" ?

**Comment by SenorRagequit** [comment, marked as answer]
Wish can be ignored - Wira doesn't have any problems anymore.
